sql/core/src/main/scala/org/apache/spark/sql/execution/python/FlatMapGroupsInBatchExec.scala

```diff
@@ -82,13 +83,14 @@ trait FlatMapGroupsInBatchExec extends SparkPlan with UnaryExecNode with PythonS
 
       val data = groupedData(iter, dedupAttributes)
 
-      val runner = new ArrowPythonRunner(
+      val runner = new GroupedArrowPythonRunner(
```

Comment:
So for a UDF using existing `Table -> Table` signature, it will be split into batches?
I guess this might cause breaking change:
suppose a UDF doing aggregation, and a group larger than `arrowMaxRecordsPerBatch`, it outputs single row before, but will generate multiples rows after this change?

Reply:
No a single group will always be a single Table in the UDF call still, the Python side will collect all batches for a stream into a Table if using the Table API